app/src/main/java/org/kiwix/kiwixmobile/di/modules/ActivityBindingModule.java

```diff
@@ -70,4 +71,8 @@
   @PerActivity
   @ContributesAndroidInjector
   public abstract HelpActivity provideHelpActivity();
+
+  @PerActivity
+  @ContributesAndroidInjector(modules = MainModule.class)
```

Comment:
MainModule is for the main package. If you need a module create one.

Reply:
Fixed. 4486c06b3c2453c4a77c78cb2a1fd00406c040cc